### PR TITLE
✨ Update .OS to .Os

### PR DESCRIPTION
**Pull request by laurentsimon**
Update .OS to .Os to match goreleaser's variable

### CODE REVIEW

File: pkg/config.go [hunk 1]
```diff
@@ -40,11 +40,13 @@ type goReleaserConfigFile struct {
 	Ldflags []string `yaml:"ldflags"`
 	Binary  string   `yaml:"binary`
 	Version int      `yaml:"version"`
+	Main    string   `yaml:"main"`
```
Comment (joshuagl) [inline pkg/config.go:43]: Seems to be a stray change from #66

File: pkg/config.go [hunk 2]
```diff
@@ -40,11 +40,13 @@ type goReleaserConfigFile struct {
...
 }
 
 type GoReleaserConfig struct {
 	Goos    string
 	Goarch  string
+	Main    string
```
Comment (joshuagl) [inline pkg/config.go:49]: Same here, stray change from #66

**Review by joshuagl — APPROVED** [review]
LGTM. Some changes to pkg/config.go from #66 are staged here. We should remove those or merge #66 first.

File: pkg/config.go (hunk 2 above)
Comment (laurentsimon) [reply pkg/config.go:49]: ouch, my bad. Auto-merge was on.

**Pull request merged**